GCTC-NTGC/gc-digital-talent
# Translation 2022-09-29

This branch brings in a round of translation.

# Notes
- I wasn't able to easily set up a French story for the homepage so I spun it off to #4371

# Testing
- Build the site, browse around in French
- Check the items from #3909
  - The third item was spun off to #4370
- Check the changes to the whitelist files to ensure they are actually equivalent words in English and French
- If you check out [Merge new translations](https://github.com/GCTC-NTGC/gc-digital-talent/commit/19c004ede322a2fcf411d144898a45957309767e) you can do a side-by-side comparison of the translation files.  The intermediate files were cleaned up in [Remove intermediate files](https://github.com/GCTC-NTGC/gc-digital-talent/commit/062a6a682ae44932ae4090b9ee7349ea87b1ba75).
- Check that the strings changed in #4307 still work.

Closes #4025
